notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1134 @@
+{
```

Comment:
<p>Why do we have this long list of features here? Are we using some columns as features and excluding other columns? </p><p> </p><p>Could you clarify why we have this list of features here, but not in https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/automl/automl_tabular_on_vertex_pipelines.ipynb?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/726/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>It's the list of all column names excluding the target column. automl_tabular_on_vertex_pipelines.ipynb has it as well, under the <code>Define training specification</code> section. I moved it to under the  <code>Setup training configuration</code> for this notebook.  </p>